src/effects/defs.h

```diff
@@ -11,6 +11,12 @@ enum class EffectEnableState {
     Enabling
 };
 
+enum class EffectBackendType {
+    BuiltIn,
+    LV2,
+    Unknown
```

Comment:
Do we need this `Unknown` value? Can we remove it?

Reply:
I added `Unknown` to handle the case of a manifest not having a backend rather than handling it while setting the effectType in `dlgprefeffects`. I think we should keep it.

Reply:
EffectManifest would always have a backend currently. But because the backendName is not set in its constructor but through `setBackendName()` function, I think we should set it to `Unknown` for the time being rather than setting it to the first value (`Built-in`) in the EffectBackendType enum.

Reply:
Oh, I thought of a use case for this Unknown value. When we parse an XML file specifying an effect chain preset and use the `EffectManifest::backendTypeFromString` function, we can use the Unknown return value to determine that the effect cannot be loaded.